discovery-infra/test_infra/controllers/node_controllers/node_controller.py

```diff
@@ -45,10 +45,6 @@ def start_all_nodes(self) -> List[Node]:
     def restart_node(self, node_name: str) -> None:
         pass
 
-    @abstractmethod
-    def format_node_disk(self, node_name: str) -> None:
```

Comment:
didn't we talked to leave this one?

Reply:
the implementation is dependent on cluster_id unfortunately

Reply:
i think better to add empty function to libvirt controller then to remove this one from interface

Reply:
Maybe let's meet in the middle and raise ``NotImplementedError``?
Instead of just passing, which is a silent error

It will still allow instantiating the class, but not using this method